README.md

```diff
@@ -113,7 +113,8 @@ parameters associated with the strategies can be configured too. By default, all
 
 The policy also includes common configuration for all the strategies:
 - `nodeSelector` - limiting the nodes which are processed
-- `evictLocalStoragePods` - allowing to evict pods with local storage
+- `evictLocalStoragePods` - allows eviction of pods with local storage
+- `evictSystemCriticalPods` - allows eviction of pods with any priority, including system pods like kube-dns
```

Comment:
It would be good to add a big, bold warning that this option should be used very carefully (and maybe a warning in the logs too)

Reply:
Sure I can add that. Also yes you correctly saw how `IsCriticalPod` needed to be split into individual functions. Also the fact that the only place that called it was evictions.go made the decision easier.